### PR TITLE
Enable RFS for font resizing

### PR DESCRIPTION
**Pull request by MartijnCuppens**
This PR only tackles the font size part of https://github.com/twbs/bootstrap/pull/29152, as [requested](https://github.com/twbs/bootstrap/pull/29152#issuecomment-527931753) by @ysds. 

- Enable RFS
- Add documentation
- Disable doc font size overrides

Preview: <https://deploy-preview-29597--twbs-bootstrap.netlify.com/>

**TODO:**

- [x] Link to versioned repo

### CODE REVIEW

**Comment by XhmikosR** [comment]
LGTM AFAICT. Feel free to squash the patches and let's wait for @mdo and @ysds .

Only thing is to see if we need to use a bigger font-size; in some case on mobile the font seems too small, but maybe I'm just not used to it 100%.

File: site/content/docs/4.3/content/typography.md [hunk 1]
```diff
@@ -272,6 +272,4 @@ Align terms and descriptions horizontally by using our grid system's predefined
 
 ## Responsive font sizes
 
-Bootstrap v4.3 ships with the option to enable responsive font sizes, allowing text to scale more naturally across device and viewport sizes. <abbr title="Responsive font sizes">RFS</abbr> can be enabled by changing the `$enable-responsive-font-sizes` Sass variable to `true` and recompiling Bootstrap.
-
-To support <abbr title="Responsive font sizes">RFS</abbr>, we use a Sass mixin to replace our normal `font-size` properties. Responsive font sizes will be compiled into `calc()` functions with a mix of `rem` and viewport units to enable the responsive scaling behavior. More about <abbr title="Responsive font sizes">RFS</abbr> and its configuration can be found on its [GitHub repository](https://github.com/twbs/rfs).
+In Bootstrap `v5`, we've enabled responsive font sizes by default, allowing text to scale more naturally across device and viewport sizes. Have a look at the [RFS]({{< docsref "/getting-started/rfs" >}}) page to find out how this works.
```
Comment (mdo) [inline site/content/docs/4.3/content/typography.md]:
```suggestion
In Bootstrap 5, we've enabled responsive font sizes by default, allowing text to scale more naturally across device and viewport sizes. Have a look at the [RFS page]({{< docsref "/getting-started/rfs" >}}) to find out how this works.
```

File: site/content/docs/4.3/getting-started/rfs.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+layout: docs
+title: RFS
+description: Bootstrap's resizing engine.
+group: getting-started
+toc: true
+---
+
+## What is RFS?
+
+Bootstrap's side project [RFS](https://github.com/twbs/rfs) is a unit resizing engine which was initially developed to resize font sizes (hence its abbreviation for Responsive Font Sizes). Nowadays RFS is capable of rescaling basically every value for any CSS property with units, like `margin`, `padding`, `border-radius` or even `box-shadow`.
```
Comment (mdo) [inline site/content/docs/4.3/getting-started/rfs.md]:
```suggestion
Bootstrap's side project [RFS](https://github.com/twbs/rfs) is a unit resizing engine which was initially developed to resize font sizes (hence its abbreviation for Responsive Font Sizes). Nowadays RFS is capable of rescaling most CSS properties with unit values like `margin`, `padding`, `border-radius`, or even `box-shadow`.
```

File: site/content/docs/4.3/getting-started/rfs.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
+---
+layout: docs
+title: RFS
+description: Bootstrap's resizing engine.
```
Comment (mdo) [inline site/content/docs/4.3/getting-started/rfs.md]:
```suggestion
description: Bootstrap's resizing engine responsively scales common CSS properties to better utilize available space across viewports and devices.
```

File: site/content/docs/4.3/getting-started/rfs.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+The mechanism automatically calculates the appropriate values based on the dimensions of the browser viewport. It will be compiled into `calc()` functions with a mix of `rem` and viewport units to enable the responsive scaling behavior.
+
+## Using RFS
+
+The mixins are included in Bootstrap and are available once you include Bootstraps' `scss`. RFS can also be installed [standalone](https://github.com/twbs/rfs#installation) if needed.
+
+### Using the mixins
+
+The `rfs()` mixin has shorthands for `font-size`, `margin`, `margin-top`, `margin-right`, `margin-bottom`, `margin-left`, `padding`, `padding-top`, `padding-right`, `padding-bottom` and `padding-left` which can be used like this:
```
Comment (mdo) [inline site/content/docs/4.3/getting-started/rfs.md]:
```suggestion
The `rfs()` mixin has shorthands for `font-size`, `margin`, `margin-top`, `margin-right`, `margin-bottom`, `margin-left`, `padding`, `padding-top`, `padding-right`, `padding-bottom`, and `padding-left`. See the example below for source Sass and compiled CSS.
```

File: site/content/docs/4.3/getting-started/rfs.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+```scss
+.title {
+  @include font-size(4rem);
+}
+```
+
+which outputs the following CSS:
```
Comment (mdo) [inline site/content/docs/4.3/getting-started/rfs.md]:
```suggestion
```

**Review by mdo — COMMENTED** [review]
Leaving some copywriting edits as suggestions here. Might make some more suggestions after testing more thoroughly. Haven't tested out too much but had a quick look on my phone and things seem to be looking good.

**Comment by XhmikosR** [comment]
Needs a squash of the last doc patches into one before merge

**Comment by MartijnCuppens** [comment]
> Only thing is to see if we need to use a bigger font-size; in some case on mobile the font seems too small, but maybe I'm just not used to it 100%.

RFS does rescale larger font sizes quite a lot, but we need this for the [display headings](https://deploy-preview-29597--twbs-bootstrap.netlify.com/docs/4.3/content/typography/#display-headings) and longer words or compounds (more present in Dutch/German).

> Haven't tested out too much but had a quick look on my phone and things seem to be looking good.

I've been testing for almost 3 years, so we should be fine 😃

**Review by mdo — APPROVED** [review]

**Review by XhmikosR — APPROVED** [review]

**Pull request merged**